guides/admin/inotify-watch-limits.md

```diff
@@ -0,0 +1,266 @@
+---
+title: Troubleshooting inotify Watcher Limit Problems
+description: Learn how to resolve problems related to the inotify Watcher Limit.
+---
+
+The [`inotify` facility] allows programs to monitor files and directories for
+changes, so that they will receive an event immediately whenever a user or
+program modifies the file or directory. Many such programs can fall back to
+periodically checking files and directories for changes, also known as polling.
+Using polling avoids the watch limit, but may result in higher system loading
+and a longer interval for the program to detect changes.
+
+`inotify` requires kernel resources (memory and processor) for each file it
+tracks. As a result, the Linux kernel limits the number of file watchers that
+each user can register. The default settings vary according to the host system
+distribution; on Ubuntu 20.04 LTS, the default limit is 8,192 watches per instance
+
+[`inotify` facility]: https://en.wikipedia.org/wiki/Inotify
+
+With some applications and tools, including Webpack or [code-server], you may
+encounter an error similar to the following:
+
+> Watchpack Error (watcher): Error: ENOSPC: System limit for number of file
+> watchers reached, watch '/some/path'
+
+[code-server]: https://github.com/cdr/code-server
+
+On a 64-bit system, each `inotify` watch that programs register will consume
+approximately 1 kB of kernel memory, which cannot be swapped to disk and is not
```

Comment:
I've always been kinda sad about ~ to denote approximate, since there's an approximate symbol ≈ or ≃ but I guess ~ is accepted enough. I'll follow whatever our general style is, and need some advice from @khorne3 on that front

Reply:
^ **Very** fair point